libs/features/mas/web-components/test/merch-card-product.test.html.js

```diff
@@ -0,0 +1,69 @@
+// @ts-nocheck
+import { runTests } from '@web/test-runner-mocha';
+import { expect } from '@esm-bundle/chai';
+
+import { mockLana } from './mocks/lana.js';
+import { mockFetch } from './mocks/fetch.js';
+
+import '../src/merch-offer.js';
+import '../src/merch-offer-select.js';
+import '../src/merch-quantity-select.js';
+
+import { appendMiloStyles } from './utils.js';
+import { mockIms } from './mocks/ims.js';
+import { withWcs } from './mocks/wcs.js';
+import mas from './mas.js';
+
+const skipTests = sessionStorage.getItem('skipTests');
+
+runTests(async () => {
+  mockIms();
+  mockLana();
+  await mockFetch(withWcs);
+  await mas();
+  if (skipTests !== null) {
```

Comment:
please remove `appendMiloStyles();` as we are in Milo now.
you can add the style in your test as at:
https://github.com/adobecom/milo/blob/stage/libs/features/mas/web-components/test/merch-card.test.html#L8